build-constraints.yaml

```diff
@@ -446,7 +446,7 @@ packages:
         - safe-foldable
 
     "Jaro Reinders <jaro.reinders@gmail.com> @Noughtmare":
-        - haskell-lsp-client
+        - haskell-lsp-client < 0
```

Comment:
this package hasn't been updated in awhile; it can be deleted if @noughtmare doesn't have any objections

Reply:
actually let's do it the other way around, noughtmare is welcome to add it back later if there's an objection :)